### PR TITLE
Feature/Processing of peer deletion

### PR DESCRIPTION
**Pull request by Milena-Czierlinski**
The `deletionStatus` provides the information about whether the peer of the Relationship is in the grace period of its Identity Deletion or was already deleted. Later, a `deletionDate` will be added that gives information about when the grace period will end or when the peer was deleted. However, this is currently not supported by the Backbone, yet.

### CODE REVIEW

File: packages/enmeshed_runtime_bridge/lib/src/javascript_handlers.dart [hunk 1]
```diff
@@ -1,15 +1,13 @@
 import 'dart:io';
 
 import 'package:device_info_plus/device_info_plus.dart';
+import 'package:enmeshed_runtime_bridge/enmeshed_runtime_bridge.dart';
```
Comment (jkoenig134) [inline packages/enmeshed_runtime_bridge/lib/src/javascript_handlers.dart:4]: wrong import.

Comment (jkoenig134) [reply packages/enmeshed_runtime_bridge/lib/src/javascript_handlers.dart:4]: the ones below were correct.

Comment (Milena-Czierlinski) [reply packages/enmeshed_runtime_bridge/lib/src/javascript_handlers.dart:4]: Oh no! 😨

**Review by jkoenig134 — APPROVED** [review]

**Comment by Milena-Czierlinski** [comment]
> The deletionStatus provides the information about whether the peer of the Relationship is in the grace period of its Identity Deletion or was already deleted. Later, a deletionDate will be added that gives information about when the grace period will end or when the peer was deleted. However, this is currently not supported by the Backbone, yet.

@aince42 FYI

**Pull request merged**